### ISSUE TITLE
Errors handling

### ISSUE THREAD

**Issue by pgalwa** [ldd/gatsby-source-github-api#19]
Hey,
could You add error handling for graphql requests? 
It's simple. I did in my fork :)

In  gatsby-node.js you have to do something like this:
```
 fetchFromGithub(url, token, graphQLQuery, variables).then(result => {
      createNode({
        data: result.data,
        id: result.id || uuid(),
        errors: result.errors || [],
```
:)

It will be very helpful for users cuz sometimes even github throws an error :)

**Comment by ldd** [comment]
Interesting proposal. I like it.

Yet, perhaps there are other parts of the result that people want?

What about something like
```
...
 fetchFromGithub(url, token, graphQLQuery, variables).then(result => {
      createNode({
        data: result.data,
        id: result.id || uuid(),
        rawResult: result,
...
```

**Comment by pgalwa** [comment]
Sounds good :) It might be helpful for debugging

**Comment by ldd** [comment]
Done. Published to npm as version `0.2.0`

Let me know if it works as intended, or if there are more changes you need.

Thanks for writing this bug report and using software I wrote.
